2020-09-09/KJEZ35-logical-turtle.md

```diff
@@ -0,0 +1,10 @@
+---
+author: KJEZ35
+layout: post
+title: "embed a tutrtle"
```

Comment:
"turtle" :)

Reply:
Thank you!

Reply:
also the file name needs to be changed as I mentioned above! other than that, looks good! can't wait to see your trinket @KJEZ35

Reply:
File name was changed, Thank you!